docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -1849,13 +1849,13 @@ Available LSF configuration options
 .. topic:: LSF_QUEUE
 
 	The name of the LSF queue you are running simulations in.
-	For e.g. ``bsub``, this option will be passed to the ``-q`` parameter:
-	https://www.ibm.com/support/knowledgecenter/en/SSETD4_9.1.3/lsf_command_ref/bsub.q.1.html
-
+	For example, ``bsub``, this option will be passed to the ``-q`` parameter:
+	https://www.ibm.com/support/knowledgecenter/SSWRJV_10.1.0/lsf_command_ref/bsub.q.1.html
+	
```

Comment:
remove whitespace

Reply:
I think this one was actually meant to be "For bsub, this option […]". There's no example here.